lib/services/locator/locator_service.dart

```diff
@@ -8,4 +11,4 @@ import 'package:survey/services/api/user/user_api_service.dart';
 
 part 'locator_service_register.dart';
 
-final locator = GetIt.instance;
+final locator = GetIt.instance..allowReassignment = true;
```

Comment:
any reason why you set `allowReassignment` here? 🙏  As I understand, this flag should only be used when we are in testing and need to override Fake class. Please correct me if I'm wrong 😄

Reply:
Yea you're right, I added this line test. 
But I'm not sure whether it's critical problem in production code? Or I can add a todo to fix this later, cause a lot of tests will need to be updated if I remove this line 😬

Reply:
@markgravity it's a redundant assignment, yes you can add a TODO comment, no risk so far if we use `get_it` correctly 🙏